qemu/tests/smbios_table.py

```diff
@@ -24,6 +24,11 @@ def run(test, params, env):
     notset_output = params.get("notset_output")
     dmidecode_exp = params.get("dmidecode_exp")
     login_timeout = float(params.get("login_timeout", 360))
+    machine_type = params.get("machine_type", "")
+    if ':' in machine_type:
```

Comment:
How about  "if "arm64" in machine_type " ?   I know, currently,  only "arm64-pci:virt" and "arm64-mmio:virt" have ":", but do we need to consider if there has other new machine type has ":" in the future?